ext/curl/lib/url.h

```diff
@@ -44,7 +44,7 @@
 #include <nrf_modem_limits.h>
 /* In embedded this needs to be a lot of smaller */
 #define UPLOADBUFFER_DEFAULT (708)
-#define UPLOADBUFFER_MAX (NRF_MODEM_IP_MAX_MESSAGE_SIZE)
+#define UPLOADBUFFER_MAX (4096)
```

Comment:
What is the actual limitation nowadays in libmodem side?
Is it CONFIG_NRF_MODEM_LIB_SHMEM_TX_SIZE? If yes, then this could be that and let it fail if no enough mem available.

Reply:
It is, probably we can use `CONFIG_NRF_MODEM_LIB_SHMEM_TX_SIZE` here then